tlslite/constants.py

```diff
@@ -107,7 +107,8 @@ class ContentType(TLSEnum):
     alert = 21
     handshake = 22
     application_data = 23
-    all = (20, 21, 22, 23)
+    heartbeat = 24
```

Comment:
please add the RFC number it comes from as a comment (see line 137 below as an example)